spec/tests/AdHelper.spec.ts

```diff
@@ -5,36 +5,30 @@ import * as Conviva from '@convivainc/conviva-js-coresdk';
 describe(AdHelper, () => {
   describe('mapAdPosition', () => {
     it('should map ad position to preroll', () => {
-      const player = {
-        getDuration: () => 100,
-      } as PlayerAPI;
+      const duration = 100 as number;
```

Comment:
Why we need the cast here?

Reply:
Removed https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/118/commits/a1f4689bf655609614c4c57cf7b56d66ded10558